docker/Dockerfile

```diff
@@ -1,5 +1,5 @@
-FROM alpine
-MAINTAINER Angus Lees <gus@inodes.org>
+FROM gcr.io/distroless/base@sha256:edc3643ddf96d75032a55e240900b68b335186f1e5fea0a95af3b4cc96020b77
```

Comment:
Please document which distroless flavor we're using, guess gcr.io/distroless/static ?

Reply:
it's using the `base` flavour (not the `static` flavour):
```
gcr.io/distroless/base@sha256....
                  ^^^^
```

I could have used the static flavour but I have other low hanging fruits

Reply:
Ok, even more reasonable to add a comment re:that IMO.

Reply:
PTAL